MMM-Sonos.js

```diff
@@ -55,12 +55,26 @@ Module.register('MMM-Sonos', {
       const roomName = this.getRoomName(item)
       if (roomName !== '') {
         const currentTrack = item.coordinator.state.currentTrack
+        let nextTrack = currentTrack
+        //Check for Spotify / Amazon Radio & Apple Music - nextTrack could be empty which is a Pain
+        if (currentTrack.uri.includes('spotify')) {
+
+        } else if (currentTrack.type !== 'radio') {
+                nextTrack = item.coordinator.state.nextTrack;
+        } else {
+                nextTrack = currentTrack
```

Comment:
```suggestion
```

Reply:
Tried it without - caused issues when jumping from one source to another where the nextTrack Version was empty

The handling from Sonos is really weird - based from what source you are comming from

Reply:
Checked it again ... you are right ... makes to sense to leave it in - adding additional failsafe for radio - checking if nextTrack is empty